hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/TestContainerManagerImpl.java

```diff
@@ -101,7 +101,7 @@ public void cleanup() throws Exception {
   }
 
   @Test
-  public void testAllocateContainer() throws Exception {
```

Comment:
nit: What's the reason for removing the public keywords here and in other test? I think they were fine.

Reply:
This is a common recommendation for Junit5 since Junit5 does not require test methods to be public. For example, there's a PMD rule [JUnit5TestShouldBePackagePrivate](https://pmd.github.io/latest/pmd_rules_java_bestpractices.html#junit5testshouldbepackageprivate).
I can revert these changes if there's a strong opinion against this practice.

Reply:
Thank you for this information, I didn't know about it. No need to revert.